deployment/scripts/install.sh

```diff
@@ -286,22 +286,22 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete imagestream,service,route --selector logging-infra=support
```

Comment:
It only deletes them if they have the label indicating it also created
them. So it won't impact dev builds (which don't have the label). The only
impact is on testing where they have to do some fiddling to get the right
tags from nonsecure registry into the IS. I don't think they'll be doing
that iteratively (but if they are, they could remove the label to prevent
it being deleted). I think it's worth it because if we don't delete them,
changes to the IMAGE_PREFIX (oops! typo!) don't get picked up in the ISs
from one run to the next.

On Mon, Apr 18, 2016 at 11:30 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployment/scripts/install.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/117#discussion_r60079663>
> :
>
> > @@ -286,22 +286,22 @@ fi
> >  echo "(Re-)Creating deployed objects"
> >  if [ "${KEEP_SUPPORT}" != true ]; then
> >  	oc process logging-support-template | oc delete -f - || :
> > -	oc delete serviceaccount,service,route --selector logging-infra=support
> > +	oc delete imagestream,service,route --selector logging-infra=support
>
> do we want to also delete image streams by default? won't that make dev
> builds more difficult?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/117/files/e4b187c1c050cdab426a760811c479128dbc0380..586c2f42e48f04a39ece6c834162112673fd9cdc#r60079663>
>